iskolivery/app/views/users/view_edit.html.erb

```diff
@@ -59,6 +58,10 @@ This work is licensed under a Creative Commons Attribution-ShareAlike 4.0 Intern
 					<%= f.label :email, text='Username or UP Mail' %>
 					<%= f.email_field :email, html: {placeholder: 'Enter username'} %>
 			    </div>
+			    <div class="ui toggle checkbox">
+				  <input type="checkbox" name="public">
```

Comment:
Instead of explicitly specifying an <input type="checkbox"> element, we use a form helper here. We use form helpers extensively in the project, to bundle similar parameters together and ensure that they're submitted all at once.

https://api.rubyonrails.org/classes/ActionView/Helpers/FormHelper.html#method-i-check_box
This can be styled as well, to do this I suggest replacing this line with a form helper.
```suggestion
				  # put the form helper here
```